ballerina/streamable_http_client.bal

```diff
@@ -39,8 +39,8 @@ public distinct isolated client class StreamableHttpClient {
     # + clientInfo - Client implementation information
     # + capabilities - Client capabilities to advertise
     # + return - `ClientError` if initialization fails, `()` on success
-    isolated remote function initialize(Implementation clientInfo, ClientCapabilities capabilities = {})
-            returns ClientError? {
+    isolated remote function initialize(Implementation clientInfo = {name: "MCP Client", version: "1.0.0"},
```

Comment:
I still thinks this should only say the major version or major + minor

Reply:
Otherwise, for MCP related patch releases we needs to do releases in our module as well

Reply:
Since we follow semver, as per semver we should always write the full form, `MAJOR.MINOR.PATCH`,
Second, this version is nothing to do with any releases, this is just a placeholder for developers to write clients without specifying the client info,